src/solve/form.ts

```diff
@@ -536,4 +537,6 @@ export async function executeRoute(
 
     onTxSuccess({}, i);
   }
+
+  trackRoute(route);
```

Comment:
We should call this on submission, so even if the user navigates away before it succeeds, we still record it

Reply:
updated